src/common/meta/Cargo.toml

```
@@ -24,7 +27,13 @@ snafu.workspace = true
 store-api = { path = "../../store-api" }
 table = { path = "../../table" }
 tokio.workspace = true
+reqwest = { version = "0.11", features = ["json"] }
+once_cell = "1.10"
 
 [dev-dependencies]
 chrono.workspace = true
 datatypes = { path = "../../datatypes" }
+hyper = { version = "0.14", features = ["full"] }
+
+[build-dependencies]
+build-data = "0.1.3"
```

hey @paomian 
I’ve noticed you are are using not the latest version of build data.
```suggestion
build-data = "0.1.4"
```
I appreciate this is a draft so more changes to come, just decided to let you know. Hope that’s ok.

---

good catch.I will change it to the latest version.